flow/flow.go

```diff
@@ -759,6 +764,14 @@ func SetStopper(IN *Flow) error {
 	return nil
 }
 
+// DealARPICMP is predefined function which will generate
+// replays to ARP and ICMP requests and automatically extract
+// corresponding packets from input flow
+// Using after merge will result to answering any of input ports
+func DealARPICMP(IN *Flow) error {
```

Comment:
If used after merge, function answers packets received on all input ports.

Also I think godoc comments require a dot at the end of sentences.